src/main/java/duke/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that starts and ends at a specific date/time.
+ * Includes a String representation of the date/time.
+ *
+ * @author Abdulelah Faisal S Al Ghrairy
+ */
+public class Event extends Task {
+
+    protected LocalDate date;
+    protected LocalTime timeBeginning;
+    protected LocalTime timeEnd;
```

Comment:
I think it is better for these fields to be `private` and provide corresponding getter methods. But very interesting idea to make a starting and ending time for `Event` 👍.